### PR TITLE
Add fr version of 15 days of Postman and Lost in Space

### PR DESCRIPTION
**Pull request by arlemi**
I've also edited a couple things I noticed as I was looking over the 15 days of Postman in english. 

- runs locally ☑️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can submit your own Quickstarts to be published on Postman's website by subm
    - Install gulp-cli `npm i -g gulp-cli`
 2. [Install Go](https://golang.org/doc/install); Homebrew installed? `brew install golang`
    - Install claat `go install github.com/googlecodelabs/tools/claat@latest`
-   - Ensure go and claat is in your `PATH` [claat path setup](#claat-related-errors)
+   - Ensure go and claat is in your `PATH` [claat path setup](#claat-related-ecrrors)
```
Comment (loopDelicious) [inline README.md:31]: If you update to `#claat-related-ecrrors` to`#user-content-1-claat-related-errors` that should work.

File: site/app/views/fr/view.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Quickstarts French",
-  "description": "Follow along with our French tutorials and step-by-step walkthroughs to get you up and running with Postman and APIs.",
+  "title": "Quickstarts Postman",
```
Comment (loopDelicious) [inline site/app/views/fr/view.json:2]: Can you revert to `Quickstarts French` this title also informs the language switcher, so it makes sense when on the main page, to switch over to "Quickstarts French" or something similar with the word "French".

**Review by loopDelicious — CHANGES_REQUESTED** [review]
@arlemi looks good - 2 changes requested

File: README.md (hunk 1 above)
Comment (arlemi) [reply README.md]: I think my imaginary cat stepped on my keyboard

**Pull request merged**